targets/TARGET_NORDIC/TARGET_NRF5x/nrf_log.h

```diff
@@ -202,28 +219,28 @@ uint32_t nrf_log_push(char * const p_str);
  * @def NRF_LOG_MODULE_REGISTER
  * @brief Macro for registering an independent module.
  */
-#if NRF_LOG_ENABLED
-
-#ifdef UNIT_TEST
-#define _CONST
-#define COMPILED_LOG_LEVEL 4
-#else
-#define _CONST const
-#define COMPILED_LOG_LEVEL NRF_LOG_LEVEL
-#endif
-#define NRF_LOG_MODULE_REGISTER()                                                             \
-    NRF_SECTION_ITEM_REGISTER(NRF_LOG_CONST_SECTION_NAME(NRF_LOG_MODULE_NAME),                \
-                            _CONST nrf_log_module_const_data_t NRF_LOG_MODULE_DATA_CONST) = { \
-            .p_module_name = STRINGIFY(NRF_LOG_MODULE_NAME),                                  \
-            .info_color_id = NRF_LOG_INFO_COLOR,                                              \
-            .debug_color_id = NRF_LOG_DEBUG_COLOR,                                            \
-            .compiled_lvl   = COMPILED_LOG_LEVEL,                                             \
-    };                                                                                        \
-    NRF_SECTION_ITEM_REGISTER(NRF_LOG_DYNAMIC_SECTION_NAME(NRF_LOG_MODULE_NAME),              \
-                          nrf_log_module_dynamic_data_t NRF_LOG_MODULE_DATA_DYNAMIC)
-#else
-#define NRF_LOG_MODULE_REGISTER() /*lint -save -e19*/ /*lint -restore*/
-#endif
+// #if NRF_LOG_ENABLED
```

Comment:
That's a lot of commented-out code!

Reply:
True, I forgot about them. Wasn't sure if they were actually needed/used in the SDK code we have.... it compiles without them though.